File: javaapps/Console/AREAPERIMETER.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
//nick buchanan
//2/17/10
//pd 8
namespace ConsoleApplication1
{
    class AREAPERIMETER
    {
        static void Main(string[] args)
        {
            Console.Write("Enter length: ");
            int length = Int32.Parse(Console.ReadLine());

            Console.Write("Enter width: ");
            int width = Int32.Parse(Console.ReadLine());

            int area = length * width;
            int perimeter = (length * 2) + (width * 2);
            Console.WriteLine("\nArea of rectangle: " + area);
            Console.WriteLine("Perimeter of Rectangle: " + perimeter);
           
        }
    }
}

File: javaapps/Console/AreaPerimeter.java

//Nicholas Larkin Buchanan
//2010-02-17
 import java.util.Scanner;

public class AreaPerimeter {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        System.out.print("Enter length: ");
        int length = Integer.parseInt(scanner.nextLine());

        System.out.print("Enter width: ");
        int width = Integer.parseInt(scanner.nextLine());

        int area = length * width;
        int perimeter = (length * 2) + (width * 2);

        System.out.println("\nArea of rectangle: " + area);
        System.out.println("Perimeter of Rectangle: " + perimeter);
    }
}
// Usage: Calculates Area and Perimeter of a rectangle given length and width.
